projects/aca-playwright-shared/src/page-objects/components/search/search-input.component.ts

```diff
@@ -23,12 +23,14 @@
  */
 
 import { Locator, Page } from '@playwright/test';
-import { BaseComponent } from '.././base.component';
+import { BaseComponent } from '../base.component';
 import { timeouts } from '../../../utils';
+import { $ } from 'protractor';
```

Comment:
this is coming from protractor and should not be used in the playwright

Reply:
I forgot to remove this, added by mistake.